.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
```

Comment:
not really related to this PR?

Reply:
Well, it is kind of related. I was blocked by Jenkins CI trying to use JDK 11, and I needed to know if we could build with JDK 11!